source/help/messaging/emoji-skin-tones.rst

```diff
@@ -0,0 +1,57 @@
+Emoji Skin Tones
+================
+
+Emoji are pictures that can communicate emotions, gestures, intent, and so much more. The Mattermost Emoji Picker uses neutral skin tones by default. You can customize the skin tone for many emoji using a native emoji picker.
+
+Selecting Emoji
+---------------
+
+Click the smile icon inside the message input box to open the Mattermost Emoji Picker.
+
+.. image:: ../../images/selectemoji.png
+  :alt: Open the Mattermost Emoji Picker.
+
+Selecting Emoji Skin Tones
+--------------------------
+
+To access alternate skin tones, use the native emoji pickers available on your platform or device.
```

Comment:
```suggestion
To access alternate skin tones, open Mattermost on your device and use the native emoji pickers provided on your platform or device.
```

Reply:
For the steps below, does the user need to be in Mattermost first? Not sure if we need to state that for all the sections or whether my suggestion will suffice? 

Also, not sure if it's something we should note but this only applies to "hand" emojis (not smileys etc) and the selection will remain as such going forward until/unless changed.